.github/workflows/lint.yaml

```
@@ -47,3 +48,16 @@ jobs:
         run: make update/web-deps
       - name: Run lint
         run: make lint/web
+  helm:
+    runs-on: ubuntu-24.04
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v3
+
+      - name: Set up Helm
+        uses: azure/setup-helm@v4
+        with:
+          version: ${{ env.HELM_VERSION }}
+
+      - name: Lint all Helm charts
+        run: helm lint ./manifests/pipecd ./manifests/piped ./manifests/helloworld ./manifests/site
```

Use the same format as other lint using `make lint/xxx` and don't hard code the directories.

```suggestion
        run: make lint/helm
```

And add the following command to the Makefile: 

```makefile
.PHONY: lint/helm
lint/helm:
	@for dir in $$(find ./manifests -mindepth 1 -maxdepth 1 -type d); do \
		helm lint $$dir || exit $$?; \
	done
```

---

sure

---

thank you,

besides, make commands can be used on local, which will be useful for developers.

---

Yes, absolutely!!